src/core/Gltf.tsx

```diff
@@ -1,67 +1,59 @@
 import * as React from 'react'
-
+import { GLTFLoader, DRACOLoader, MeshoptDecoder } from 'three-stdlib'
+import { useLoader } from '@react-three/fiber'
 import { Clone } from './Clone'
-import { type Loader } from 'three'
-import { type GLTF, GLTFLoader, DRACOLoader, MeshoptDecoder } from 'three-stdlib'
-import { type ObjectMap, useLoader } from '@react-three/fiber'
 
 let dracoLoader: DRACOLoader | null = null
-
 let decoderPath: string = 'https://www.gstatic.com/draco/versioned/decoders/1.5.5/'
 
-function extensions(useDraco: boolean | string, useMeshopt: boolean, extendLoader?: (loader: GLTFLoader) => void) {
-  return (loader: Loader) => {
+type Path = string | string[]
+type UseDraco = boolean | string
+type UseMeshopt = boolean
+type ExtendLoader = (loader: GLTFLoader) => void
+
+function extensions(useDraco: UseDraco = true, useMeshopt: UseMeshopt = true, extendLoader?: ExtendLoader) {
+  return (loader: GLTFLoader) => {
```

Comment:
`: GLTFLoader` type-assertion instean of `: Loader` => it avoid multiple `as GLTFLoader` later in the function